packages/ui/packages/Carousel/src/demo.js

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import { useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { Slider } from './core/index';
+import { StyledSimpleCarousel } from './core/styles';
+import { StyledCarouselDemo } from './core/styles.demo';
+
+const SimpleCarouselDemo = props => {
+  const {
+    className,
+    children,
+    slideConfig,
+    showingSlide,
+    refresh,
```

Comment:
`demo.js` should not be a part of /src path, as it will compile with babel and publish to npm